test/TestTransmuteDPOS.spec.js

```diff
@@ -347,6 +344,12 @@ contract('TransmuteDPOS', accounts => {
       assert.equal(newAmount, (await tdpos.getProvider(accounts[0]))[0].toNumber()); // [0] is totalAmountBonded
```

Comment:
comment is good, but a variable would be better.

Reply:
Agreed, replaced comments with variable names